marklogic-data-hub-central/ui/e2e/cypress/integration/explore/entitiesSidepanel.spec.tsx

```diff
@@ -144,6 +148,8 @@ describe("Test '/Explore' left sidebar", () => {
     entitiesSidebar.selectBaseEntityOption("Order");
     entitiesSidebar.getBaseEntityOption("Order").should("be.visible");
     cy.wait(1000);
+    browsePage.getStagingDatabaseButton().click();
```

Comment:
Why are we clicking on both buttons?

Reply:
Definitely weird. Didn't see that before. Removed the first click.